index.html

```diff
@@ -5,27 +5,20 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="stylesheet" type="text/css" href="">
     <title>Project Ziejek</title>
 </head>
 <body>
     <header class="header__main">
         <nav class="header__nav">
             <ul class="header__buttons">
-                <li class="header__button">
-                    <a href="">Home</a>
-                </li>
-                <li class="header__button">
-                    <a href="">About Us</a>
-                </li>
+                <a class="header__button" href="">Home</a>
+                <a class="header__button" href="">About Us</a>  
             </ul>
-            <img class="header__logo" src="" alt="">
+            <p class="header__logo">Univerz university</p>
```

Comment:
Tutaj powinno być logo jako zdjęcie + link do strony głównej

Reply:
Wiem ale nie znalazłem żadnego takiego Logo i nie miałem nic do modyfikowania przez to więc sobie zrobiłem <p> ( tak jest na tym projekcie jako tekst po prostu )